ext/mixed/js/text-scanner.js

```diff
@@ -299,11 +299,11 @@ class TextScanner {
             }
 
             const textSource = docRangeFromPoint(x, y, this.options.scanning.deepDomScan);
-            if (this.textSourceCurrent !== null && this.textSourceCurrent.equals(textSource)) {
-                return;
-            }
-
             try {
+                if (this.textSourceCurrent !== null && this.textSourceCurrent.equals(textSource)) {
```

Comment:
If functionality is affected, it could also be changed here if that makes more sense code wise

Reply:
There's only two places in the codebase that `equals` is used, and there wasn't anything apparent that this change would break. The function is still performing the equality check, the only difference is that if an imposter is used, the imposter is allowed to be a different element since the original is removed.

The only case where this might be a bit odd is if you scan an input, that input changes, and then the user scans the same _position_ of that input. The text at the location would be different, but `equals` would return `true`. Technically this is also the case for text elements, but those are arguably less likely to change. In both cases, the side effects of this rare case would just be that the user has to move the scanning cursor somewhere else (±1 character) and then return to the original position.

Reply:
> The only case where this might be a bit odd is if you scan an input, that input changes, and then the user scans the same position of that input. The text at the location would be different, but equals would return true. Technically this is also the case for text elements, but those are arguably less likely to change. In both cases, the side effects of this rare case would just be that the user has to move the scanning cursor somewhere else (±1 character) and then return to the original position.

True, didn't think of that. Shouldn't be a big issue as you said.